### ISSUE TITLE
error running hashcat on Git Version

### ISSUE THREAD

**Issue by HaxorGaruda** [hashcat/hashcat#1499]
I got This error when running on master git version : 
```
clEnqueueWriteBuffer(): CL_INVALID_COMMAND_QU
clEnqueueWriteBuffer(): CL_INVALID_COMMAND_QU
clEnqueueWriteBuffer(): CL_INVALID_COMMAND_QUEUE
```
but when compile release version all is Ok ... 
thanks ... 

my Os : Parrot Os 
Nvidia-cuda-toolkit : 9.0 
nvidia GTX 960M
running on Optimus Mode ( optirun )

thanks ..

**Comment by jsteube** [comment]
I can't reproduce on my systems. 

Please checkout 7a8239b4c1bb72ec963c841a1a53d96848cc547f and see if it still works on yours.

**Comment by HaxorGaruda** [comment]
Thanks .. all Fix, before i use cuda 8.0 and after upgrade to cuda 9.0 not work, just remove all, and Re clone git repo.. All done..

**Comment by jsteube** [comment]
OK, thanks for update!
